userdocs/diagnostics/multiple-class-inheritance.md

```diff
@@ -0,0 +1,37 @@
+
+# Multiple Inheritance of Classes
+In some programming languages, object can inherit features and characteristics from multiple parent or base classes. This feature is called multiple inheritence and is unsupported in swift. However, Swift provide language features that can acheive results similar to multiple inheritence through the use of protocols.
+
+For example, the protocol `Animal` and `Flyable` defines blueprints of methods like `move()`. The class `Bird` now provide concrete implementation to its conforming protocols.
+
+```swift
+protocol Animal {
+    func move()
+}
+
+class Bird: Animal, Flyable {
+    func move() {
+        print("Moving...")
+    }
+}
+
+```
```

Comment:
Could you add the definition of `Flyable` to the code example? Without it, it's less clear what you're trying to show here.

Reply:
Done.